auction_seller/models/seller_board.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2022 Quartile Limited
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+
+
+class SellerBoard(models.Model):
+    _name = "seller.board"
+
+    name = fields.Char()
+    state = fields.Selection(
```

Comment:
I think this field name `state` is misleading.  How did we pick this field name?  I think something like `board_type` would be better.

Reply:
I used sst V11 dashboard as a reference.   `state` was used on that dashboard.